server/webhook_jira.go

```diff
@@ -10,6 +10,47 @@ import (
 	"github.com/andygrunwald/go-jira"
 )
 
+const (
+	eventCreatedStr            = "event_created"
```

Comment:
These are all suffixed with "Str" to avoid name conflicts with the original masks used [here](https://github.com/mattermost/mattermost-plugin-jira/blob/a5de6539fbf5749a431e57a1f90bbf37e50fa1e2/server/webhook_http.go#L23).

Reply:
Can we clean it up so we have only one set?

Reply:
@crspeller For sure, will include in this PR